python/modem-firmware-1.3+/command_interface.py

```diff
@@ -96,7 +98,8 @@ def at_command(self, at_command, wait_for_result=False):
         """Write an AT command to the command interface. Optionally wait for OK"""
 
         # AT commands are written directly as-is with the ATCommandInterface:
-        self.write_raw(at_command)
+        at_cmd_prefix = 'at ' if self.shell else ''
+        self.write_raw(f'{at_cmd_prefix}{at_command}')
```

Comment:
This can stay the same, `self.shell` just needs to become a member of `ATCommandInterface` directly, rather than live in the superclass

Reply:
This is already solved in `format_cred()` in `device_credentials_installer.py`.

Reply:
Ah, my bad, I guess I forgot a solution existing

Reply:
Wait, is this actually solved by that?

All it does is replace `\n` with `\\n`. Does the Zephyr shell support that? I didn't think that it did?

Reply:
Yep -- I did this back on the gateway project, and tested it again yesterday with the provisioning shell.

Reply:
I forgot a change was required on the at shell command. Opened a PR for that.